frontend2/src/components/tables/submissions/TourneySubTable.tsx

```diff
@@ -15,7 +15,9 @@ const TourneySubTable: React.FC<TourneySubTableProps> = ({ data, loading }) => {
     <Table
       data={data ?? []}
       loading={loading}
-      keyFromValue={(match) => match.id.toString()}
+      keyFromValue={(match) =>
+        match.tournament.toString() + match.id.toString()
+      }
```

Comment:
The recommendation below is so that the encoding is unambiguous (note that match ids should always be a number)
```suggestion
      keyFromValue={(match) =>
        match.tournament.toString() + "match" + match.id.toString()
      }
```